Content.Shared/Inventory/InventorySystem.Equip.cs

```diff
@@ -102,6 +159,28 @@ public virtual bool TryEquip(EntityUid actor, EntityUid target, EntityUid itemUi
         return true;
     }
 
+    public bool CanAccess(EntityUid actor, EntityUid target, EntityUid itemUid)
+    {
+        // Can the actor reach the target?
+        if (actor != target && !( actor.InRangeUnobstructed(target) && actor.IsInSameOrParentContainer(target)))
```

Comment:
Does containersystem have equivalents for IsInSameOrParentContainer?

Reply:
Not AFAIK. but I can open a PR to add it.

Reply:
Ill add this and replace `IsInSameOrParentContainer` usages in this and other systems in a separate PR